code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -366,7 +366,7 @@
 	if(M.chem_doses[type] == metabolism * threshold)
 		M.confused += 2
 		M.drowsyness += 2
-	else if(M.chem_doses[type] < 2 * threshold)
+	else if(M.chem_doses[type] < 1.5 * threshold)
```

Comment:
rebalancing this should probably be done by editing threshold rather than the multiplier on it. can you explain this change?

Reply:
I'm not sure what you mean with this. The modifier is what technically handles it, based on the threshold. This is why editing the individual values is a saner, safer way to do so than editing the threshold which is a mastervalue for all of them.

These are minor edits.